README.md

```diff
@@ -514,7 +514,7 @@ On the command line, options can be given as:
 - `-ffilename` (no space required)
 - `-abcf filename` (flags and option can be combined)
 - `--long` (long flag)
-- `--long_flag=true` (long flag with equals to override default value)
+- `--long_flag=true` (long flag with equals -- to override default value)
```

Comment:
this is a little different from most changes and is very manual (all changes are manual, but this is more so). I'm happy to drop it or anything else, but I think it kinda helps to clarify that the second part describes the first.